app/views/conference_registrations/show.html.haml

```diff
@@ -89,29 +89,27 @@
           %span.fa-stack
             %i.fa.fa-square-o.fa-stack-2x
             %i.fa.fa-ticket.fa-stack-1x
-          Tickets
+          Ticket Purchases
```

Comment:
Well we don't allow registrations with no tickets. We will allow free tickets to allow for this (https://github.com/openSUSE/osem/pull/1118).